views/pages/toplevel/jobs.njk

```diff
@@ -38,6 +50,38 @@
             <a href="{{ buildUrl('toplevel', 'benefits') }}" class="btn btn--small">{{ copy.sections.benefits.postscript }}</a>
         </div>
 
+        {# Current Vacancies #}
+        {% if copy.sections.currentVacancies %}
+        <div class="content-box content-box--borderless inner--wide-only accent--blue a--border-top" id="current-vacancies">
```

Comment:
Could you indent this inside the `{% if ... %}` block? I know it's a big chunk but always aids readability IMO